egs/sre16/v1/local/nnet3/xvector/prepare_feats_for_egs.sh

```diff
@@ -13,6 +13,7 @@ norm_vars=false
 center=true
 compress=true
 cmn_window=300
+write_utt2num_frames=false  # if true writes utt2num_frames
```

Comment:
Is there any circumstance under which you *don't* need this option? It seems to me it would be better to not have an option at all, and just write this in any case.